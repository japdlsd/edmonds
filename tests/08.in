30 200
1 3 8
1 5 8
1 6 10
1 11 6
1 17 5
1 18 2
1 19 2
1 21 5
1 22 4
1 24 3
1 26 4
1 27 5
1 28 3
1 29 4
2 6 6
2 9 5
2 12 7
2 24 4
3 5 4
3 7 1
3 8 10
3 9 2
3 10 6
3 11 10
3 12 9
3 15 9
3 16 9
3 18 10
3 19 4
3 22 2
3 23 10
3 26 3
4 5 10
4 7 1
4 8 9
4 10 4
4 11 2
4 12 5
4 13 9
4 16 3
4 17 3
4 18 6
4 20 2
4 21 8
4 25 8
4 28 2
4 29 10
5 6 2
5 8 1
5 9 5
5 11 2
5 13 2
5 14 5
5 16 10
5 21 9
5 23 1
5 24 8
5 27 1
5 30 2
6 7 6
6 9 8
6 11 6
6 14 9
6 16 5
6 17 7
6 22 10
6 23 7
6 24 4
6 25 1
6 27 4
6 29 5
6 30 7
7 9 4
7 10 2
7 12 6
7 13 2
7 14 9
7 15 7
7 17 5
7 19 4
7 21 8
7 22 7
7 23 6
7 24 9
7 26 8
7 27 2
7 29 3
8 9 3
8 13 5
8 16 5
8 18 9
8 21 1
8 24 3
8 30 5
9 10 2
9 11 6
9 13 4
9 15 1
9 17 2
9 19 3
9 22 7
9 24 9
9 28 10
9 29 2
10 13 3
10 14 4
10 15 5
10 17 3
10 18 5
10 19 10
10 21 10
10 23 9
10 24 3
10 27 10
10 29 5
10 30 2
11 13 8
11 16 2
11 17 3
11 18 10
11 23 2
11 24 9
11 26 2
11 29 1
12 14 2
12 15 6
12 16 7
12 19 5
12 20 8
12 21 4
12 22 2
12 23 9
12 24 10
12 25 1
12 26 5
12 27 3
13 14 1
13 20 8
13 21 10
13 24 5
13 29 9
13 30 4
14 16 5
14 19 2
14 21 6
14 26 7
14 29 8
15 16 6
15 18 3
15 20 2
15 24 7
15 27 6
15 28 5
15 29 2
16 18 7
16 20 1
16 21 4
16 26 5
16 27 6
16 29 6
16 30 4
17 18 4
17 25 4
17 28 8
18 23 4
18 27 6
18 28 6
19 20 5
19 21 4
19 22 3
19 26 1
19 28 5
19 29 2
19 30 10
20 23 9
20 24 3
20 25 1
20 29 9
21 22 1
21 23 6
21 24 6
21 25 6
21 27 6
21 28 5
22 28 6
22 29 6
22 30 8
23 26 9
24 25 4
24 27 9
24 30 9
25 27 3
25 28 1
25 29 4
25 30 1
26 30 10
27 29 7
27 30 1
28 29 1
28 30 10
