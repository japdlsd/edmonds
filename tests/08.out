23
27 30
19 26
16 20
13 14
12 22
11 29
10 17
9 15
8 21
6 25
5 23
4 28
3 7
2 24
1 18

